app/src/components/trace/LatencyText.tsx

```diff
@@ -39,6 +39,7 @@ export function LatencyText({
       alignItems="center"
       justifyContent="start"
       gap="size-50"
+      className="latency-text"
```

Comment:
Where are these class names coming from? Why not emotion?

Reply:
the emotion class names are generated so unstable. I'm currently just trying to just avoid using resize observers but maybe if it doesn't work out we switch.